provider/ecs/ecs_test.go

```diff
@@ -343,8 +343,41 @@ func TestTaskChunking(t *testing.T) {
 			outCount = append(outCount, len(el))
 		}
 
-		if !reflect.DeepEqual(outCount, c.expectedLengths) {
-			t.Errorf("Chunking %d elements, expected %#v, got %#v", c.count, c.expectedLengths, outCount)
+		if !reflect.DeepEqual(outCount, test.expectedLengths) {
+			t.Errorf("Chunking %d elements, expected %#v, got %#v", test.count, test.expectedLengths, outCount)
 		}
 	}
 }
+
+func TestEcsGetBasicAuth(t *testing.T) {
+	cases := []struct {
+		desc     string
+		instance ecsInstance
+		expected []string
+	}{
+		{
+			desc:     "label missing",
+			instance: simpleEcsInstance(map[string]*string{}),
+			expected: []string{},
+		},
+		{
+			desc: "label existing",
+			instance: simpleEcsInstance(map[string]*string{
+				types.LabelFrontendAuthBasic: aws.String("user:password"),
+			}),
+			expected: []string{"user:password"},
+		},
+	}
+
+	for _, test := range cases {
+		c := test
```

Comment:
`test := test` would work as well. No need to change the variable name.

Reply:
You right. A refactoring of ECS provider has been done in #2050. I will do the change to be consistent with this refactoring

Reply:
👍